pkg/aws/ecr/ecr.go

```diff
@@ -0,0 +1,72 @@
+package ecr
+
+import (
+	"fmt"
+
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/ecr"
+)
+
+type ECR interface {
+	TagImage(repositoryName, currentTag, newTag *string) error
+}
+
+type ECRImpl struct {
+	svc *ecr.ECR
+}
+
+func (e ECRImpl) TagImage(repositoryName, currentTag, newTag *string) error {
```

Comment:
I thinking if this shoudn't be `ReplaceTag` (and same in command), but from what I see aws-sdk does not allow for setting multiple tags

Reply:
it does not replace the tag but adds another one. You can check latest tags for resume-points.

Reply:
there are two options how to get the image, by some existing tag or by image SHA.
Since in drone  pipeline  images are tagged with 10 chars from commit SHA and commit SHA is available for all steps in pipeline i'm using it as "currentTag"